tests/acceptance/features/loginUsingEmailAddress.feature

```diff
@@ -0,0 +1,41 @@
+@insulated @disablePreviews
+Feature: login users
+
+As an admin
+I want users to login with their email address stored on a LDAP server
+So that they only need to remember one username and password (SSO)
+
+	Scenario: login with default settings
+		When the LDAP users have been resyncted
+		Then it should be possible to login with the username "user1@example.org" and password "1234" into the WebUI
+
+	Scenario: using ldap filter including email field 
+		When the LDAP config "LDAPTestId" has these settings:
+		|key                | value                                                                                    |
+		|ldapLoginFilter    | (&(\|(objectclass=inetOrgPerson))(\|(uid=%uid)(\|(mailPrimaryAddress=%uid)(mail=%uid)))) |
+		|ldapEmailAttribute |                                                                                          |
+		And the LDAP users have been resyncted
+		Then it should be possible to login with the username "user1@example.org" and password "1234" into the WebUI
+
+	Scenario: using ldapEmailAttribute but loginFilter lacks email field 
+		When the LDAP config "LDAPTestId" has these settings:
+		|key                | value                                            |
+		|ldapLoginFilter    | (&(\|(objectclass=inetOrgPerson))(\|(uid=%uid))) |
+		|ldapEmailAttribute | mail                                             |
+		And the LDAP users have been resyncted
+		Then it should be possible to login with the username "user1@example.org" and password "1234" into the WebUI
+
+	Scenario: no ldapEmailAttribute and loginFilter lacks email field 
+		When the LDAP config "LDAPTestId" has these settings:
+		|key                | value                                            |
+		|ldapLoginFilter    | (&(\|(objectclass=inetOrgPerson))(\|(uid=%uid))) |
+		|ldapEmailAttribute |                                                  |
+		And the LDAP users have been resyncted
+		Then it should not be possible to login with the username "user1@example.org" and password "1234" into the WebUI
+
+	Scenario: change Email addess on LDAP server
+		When the admin sets the ldap attribute "mail" of the entry "uid=user1,ou=TestUsers" to "user1-change@example.org"
+		And the LDAP users have been resyncted
+		Then it should not be possible to login with the username "user1@example.org" and password "1234" into the WebUI
+		But it should be possible to login with the username "user1-change@example.org" and password "1234" into the WebUI
```

Comment:
Well, only if the login fiter contains the mail / mailPrimaryAddress / _insert ldap attribute containing the mail_